index.js

```diff
@@ -69,12 +69,16 @@ var SpecReporter = function(baseReporterDecorator, formatError, config) {
     return (function(browser, result) {
       var suite = result.suite;
       var indent = "  ";
+      var browserName = browser.name.split(' (')[0];
+      if (this.USE_COLORS) {
+        browserName = browserName.cyan;
```

Comment:
Could you point me to where this value exists? I'm not familiar with this. Shouldn't this be configurable to accept more than just the cyan browser and be able to take other colors?